crates/rpc/src/method/simulate_transactions.rs

```diff
@@ -865,1310 +865,6 @@ pub(crate) mod tests {
                 }
             }
 
-            pub fn declare_without_fee_transfer(
```

Comment:
Do we want to replace the building logic with fixed fixtures? The way the test data is derived is informative - at least potentially... :-)

Reply:
Those fixtures were just for output comparison, they're no longer relevant. Let me know if I'm not understanding your point properly.

Reply:
Still not sure what you're suggesting here...

Reply:
Leave the tests unchanged, basically...